presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -837,23 +834,32 @@ protected Void visitCreateTable(CreateTable node, Integer indent)
                 builder.append("\nCOMMENT " + formatStringLiteral(node.getComment().get()));
             }
 
-            builder.append(formatProperties(node.getProperties()));
+            builder.append(formatProperties(node.getProperties(), true));
 
             return null;
         }
 
-        private String formatProperties(List<Property> properties)
+        private String formatProperties(List<Property> properties, boolean multiLine)
```

Comment:
I'm thinking this would be easier to read if we have two methods `formatPropertiesOneLine` and `formatPropertiesMultiLine`. There will be a bit of code duplication, but each variant will be simpler.